src/components/budget/budget.service.ts

```diff
@@ -168,20 +163,16 @@ export class BudgetService {
 
     const result = await this.budgetRepo.readOne(id, session, view);
 
-    const perms = await this.authorizationService.getPermissions({
-      resource: Budget,
-      sessionOrUserId: session,
-      dto: result as ResourceShape<Budget>['prototype'],
-    });
+    const privs = this.privileges.for(session, Budget, result).all;
 
     const securedProps = parseSecuredProperties(
       result,
-      perms as PermissionsOf<Budget>,
+      privs,
       Budget.SecuredProps,
     );
```

Comment:
Can we drop this for
```ts
privileges.secure(result)
```
It could replace the final return spread (except for `status` and `records`)